src/labels/CustomLabels.labels

```diff
@@ -5163,7 +5163,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>CallableApi Job.Schedule Invalid Class</shortDescription>
-        <value>An Apex class for &quot;{0}&quot; cannot be found or does not implement the Scheduleable interface</value>
+        <value>The Apex class {0} doesn''t exist or doesn''t implement the Schedulable interface.</value>
```

Comment:
@npsp-reedestockton  The apostrophe's need to be escaped, though why are there two?

Reply:
Actually, in my testing, escaping the apostrophe does not work in a context where the error is logged.  "\'" and &apos; both result in "doesnt".  

The double apostrophe is what actually makes the contraction appear correctly.

Reply:
The perfect compromise.  Double escaped apostrophes.  ;-)